streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -136,15 +137,24 @@ private void configurePartitionAssignor(final Map<String, Object> props) {
         partitionAssignor.configure(configurationMap);
     }
 
-    private void mockTaskManager(final Set<TaskId> prevTasks,
-                                 final Set<TaskId> cachedTasks,
-                                 final UUID processId,
-                                 final InternalTopologyBuilder builder) {
+    private void createMockTaskManager() {
+        final StreamsBuilder builder = new StreamsBuilder();
+        final InternalTopologyBuilder internalTopologyBuilder = TopologyWrapper.getInternalTopologyBuilder(builder.build());
+        internalTopologyBuilder.setApplicationId(applicationId);
+
+        createMockTaskManager(emptyTasks, emptyTasks, UUID.randomUUID(), internalTopologyBuilder);
+    }
+
+    private void createMockTaskManager(final Set<TaskId> prevTasks,
+                                       final Set<TaskId> cachedTasks,
+                                       final UUID processId,
+                                       final InternalTopologyBuilder builder) {
+        taskManager = EasyMock.createNiceMock(TaskManager.class);
+        EasyMock.expect(taskManager.adminClient()).andReturn(null).anyTimes();
         EasyMock.expect(taskManager.builder()).andReturn(builder).anyTimes();
         EasyMock.expect(taskManager.prevActiveTaskIds()).andReturn(prevTasks).anyTimes();
         EasyMock.expect(taskManager.cachedTasksIds()).andReturn(cachedTasks).anyTimes();
         EasyMock.expect(taskManager.processId()).andReturn(processId).anyTimes();
-        EasyMock.replay(taskManager);
```

Comment:
We sometimes add more expectations to the mock before using it, so I've moved the `replay` inline, so each method can customize their task manager.